.travis.yml

```diff
@@ -14,4 +14,13 @@ script:
   - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "virtualenv /tmp && test -f /tmp/bin/pip"
   - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "virtualenv -ppython3.5 /tmp && test -f /tmp/bin/pip"
   - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "python3.6 -m venv /tmp && test -f /tmp/bin/pip"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "virtualenv-2.7 /tmp && /tmp/bin/pip -vvv install cryptography" 
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "python3.5 -m venv /tmp && /tmp/bin/pip -vvv install cryptography" 
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "python3.6 -m venv /tmp && /tmp/bin/pip -vvv install cryptography" 
   - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "docker-compose --version"
+  - docker run --rm kpndigital/tox:py27_py35_py36 bash -c "printf '[tox]\\nskipsdist=true\\n[testenv]\\ndeps=pytest\\ncommands=py.test --version\\n' > tox.ini && pip install tox && tox -e py27"
```

Comment:
Is not it better to ship tox with the image?

pytest.ini could be provided with the source code and simply mounted into container FS

Reply:
we currently install tox in all make files. so the image would be the only environment where tox is globally installed.